code/modules/reagents/chemistry/machinery/reagentgrinder.dm

```diff
@@ -112,17 +112,17 @@
 	if (istype(I, /obj/item/reagent_containers) && !(I.item_flags & ABSTRACT) && I.is_open_container())
 		if (!beaker)
 			if(!user.transferItemToLoc(I, src))
-				to_chat(user, "<span class='warning'>[I] is stuck to your hand!</span>")
+				to_chat(user, "<span class='warning'>\The [I] is stuck to your hand!</span>")
```

Comment:
These changes are unnecessary, `[x]` automatically becomes `\the [x]` or `\The [x]` depending on context.

Reply:
If grinder name isn't capitalized, the line won't be capitalized. To solve this in a way that matches other usage in the code, I added a The.